packages/gestalt-core/build.js

```diff
@@ -57,7 +57,15 @@ const cssModules = (options = {}) => {
         scopeNames[hash] = classnameBuilder.getMinifiedClassname(hash);
       }
 
-      return scopeNames[hash];
+      // if it's dev mode, append the full class name for debugging and don't minify classnames
```

Comment:
Let's make this change in a separate PR. This isn't specific to adding this new component.